analyzer/src/funTest/assets/projects/synthetic/spdx/project/project.spdx.yml

```diff
@@ -22,6 +22,7 @@ packages:
   licenseDeclared: "Apache-2.0 AND curl AND LicenseRef-Proprietary-ExampleInc"
   name: "xyz"
   versionInfo: "0.1.0"
+  originator: "Person: Thomas Steenbergen"
```

Comment:
Is there a reason you added the originator for package xyz and openssl and not for curl?

Reply:
I missed that. Updated it now to contain curl as well